src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs

```diff
@@ -110,16 +112,50 @@ private async void FileTagItem_ItemClick(object sender, ItemClickEventArgs e)
 				await itemViewModel.ClickCommand.ExecuteAsync(null);
 		}
 
-		private async void Item_RightTapped(object sender, RightTappedRoutedEventArgs e)
+		private void TagTitle_RightTapped(object sender, RightTappedRoutedEventArgs e)
+		{
+			if (sender is not Grid grid || grid.Parent is not Grid parent)
+				return;
+
+			LoadContextMenuItem(parent, e, true);
```

Comment:
Would it be possible to create the menu in xaml?

Reply:
Yes, anyway I'll mark this as Draft and I'll change this once we have a <em>final</em> context, to avoid working twice